.github/workflows/validate-parent.yml

```diff
@@ -0,0 +1,64 @@
+name: Validate parent of fork
+
+on:
+  issues:
+    types: [opened]
```

Comment:
We have a lot of workflows running on "issue: opened", can we make a signal chain or something?

Reply:
Why? You run each workflow in isolation for a reason, so they need to trigger for a reason as well.